crates/miden-lib/src/account/auth/mod.rs

```diff
@@ -142,8 +199,12 @@ mod tests {
         let public_key_slot = account.storage().get_item(0).expect("storage slot 0 access failed");
         assert_eq!(public_key_slot, Word::from(public_key));
 
-        let num_procs_slot = account.storage().get_item(1).expect("storage slot 1 access failed");
-        assert_eq!(num_procs_slot, Word::empty());
+        let slot_1 = account.storage().get_item(1).expect("storage slot 1 access failed");
+        // Slot 1 should be [num_tracked_procs, allow_unauthorized_output_notes,
+        // allow_unauthorized_input_notes, 0] With 0 procedures,
+        // allow_unauthorized_output_notes=false, and allow_unauthorized_input_notes=false, this
+        // should be [0, 0, 0, 0]
+        assert_eq!(slot_1, Word::empty());
```

Comment:
Nit: Should we put the sentence starting with "With 0 procedures" on its own line for readability?